pkg/controller/seed-controller-manager/mla/mla.go

```diff
@@ -112,6 +113,9 @@ func Add(
 	if err := newAlertmanagerReconciler(mgr, log, numWorkers, workerName, versions, httpClient, cortexAlertmanagerURL, mlaEnabled); err != nil {
 		return fmt.Errorf("failed to create mla alertmanager configuration controller: %v", err)
 	}
+	if err := newUserGrafanaReconciler(mgr, log, numWorkers, workerName, versions, grafanaClient, httpClient, grafanaURL, grafanaHeader, mlaEnabled); err != nil {
+		return fmt.Errorf("failed to create mla userprojectbinding controller: %v", err)
```

Comment:
Can you please re-format a bit of the error message? I guess this is not `mla userprojectbinding controller`, also can you use `%w` instead of `%v`?

Reply:
Also can you please refresh the comment of this `Add` function to make it up to date?

Reply:
yeah, you are right, thx.
Done